sound/soc/sof/ipc4-topology.c

```diff
@@ -1380,6 +1380,9 @@ static int sof_ipc4_update_hw_params(struct snd_sof_dev *sdev, struct snd_pcm_hw
 			case SOF_IPC4_TYPE_UNSIGNED_INTEGER:
 				snd_fmt = SNDRV_PCM_FORMAT_U8;
 				break;
+			case SOF_IPC4_TYPE_SIGNED_INTEGER:
+				snd_fmt = SNDRV_PCM_FORMAT_S8;
+				break;
```

Comment:
Fixes: c04c2e829649 ("ASoC: SOF: ipc4-topology: Add support for 8-bit formats")

Reply:
Is this kind of patch considered as fix? It's a new format supported. Though I missed this from an internal requirement that wasn't very specific.

I can add that line yes, if it's right for this case.